src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4947,4 +4947,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_OutVarDeconstructionIsNotSupported" xml:space="preserve">
     <value>Deconstruction is not supported for an 'out' argument.</value>
   </data>
+  <data name="ERR_ExpressionVariableInConstructorOrFieldInitializer" xml:space="preserve">
+    <value>Out variable or pattern variable declarations are not allowed within constructor/field/auto-implemented property initializers.</value>
```

Comment:
Also, I believe the first "or" should be "and" (they are *both* forbidden), so I suggest

> Out variable and pattern variable declarations are not allowed within constructor initializers, field initializers, or property initializers.

This is long-winded enough that you could consider specializing it.

---
In reply to: [77212251](https://github.com/dotnet/roslyn/pull/13510#discussion_r77212251) [](ancestors = 77212251)